src/components/ModalPin.js

```diff
@@ -129,7 +136,7 @@ export class ModalPin extends React.Component {
   }
 }
 
-export default ModalPin;
+export default connect(mapStateToProps)(ModalPin);
```

Comment:
Same here, should we just pass wallet as a prop from the screen rendering this component?